localized/en-us/games/eggquiz/eggquizmath_levels.tsv

```diff
@@ -181,8 +180,8 @@ PostTest_7	1	7001, 7002~7003, 7004, 7005, 7006~7007, 7008, 7009, 7010~7011, 7012
 			7009	operations_drag	11~50-11~30=?	regroupingfalse	vertical			1~50				
 			7010	word_problem	SÅñâè Ùð âéÕã 5 ×ÀçÜØæ¡ Â·UUUUÇ¸è¢Ð ©âÙð ÎôÂãÚ ×ð¢ 4 ¥æñÚ ×ÀçÜØæ¡ Â·UUUUÇ¸è¢Ð SÅñâè Ùð ·éUUUÜ ç·UUUUÌÙè ×ÀÜè Â·UUUUÇ¸è?	स्टैसी ने सुबह ५ मछलियाँ पकड़ीं। उसने दोपहर में ४ और मछलियाँ पकड़ीं। स्टैसी ने कुल कितनी मछली पकड़ी?				9				
 			7011	word_problem	Íæò×â ·ðUUU ¹ðÌ ×ð¢ 2 Õ·UUUUçÚØæ¢ ãñ¢Ð ©â·ðUUU ¹ðÌ ÂÚ Õ·UUUUçÚØô¢ ·UUUè ÌéÜÙæ ×ð¢ 6 ¥çÏ·UUUU ×éç»üØæ¢ ãñ¢Ð ©â·ðUUU Âæâ ç·UUUUÌÙè ×éç»üØæ¢ ãñ¢?	हेनरी के पास ४ डाक टिकट हैं। एरिक के पास हेनरी की तुलना में ५ अधिक डाक टिकट हैं। एरिक के पास कितने डाक टिकट हैं?				8				
-			7012	word_problem	°·UUUU àææ¹æ ÂÚ 4 Âÿæè ÕñÆð ãñ¢Ð 1 ©Ç¸ »ØæÐ àææ¹æ ÂÚ ç·UUUUÌÙð Âÿæè Õ¿ð ãñ¢?	एक पेड़ की शाखा पर ८ पक्षी बैठे। कुछ उड़ गए। अब शाखा पर ३ पक्षी हैं। कितने पक्षी उड़ गए?				3				
-			7013	word_problem	¥æÂ·ðUUU Âæâ 7 »éyÕæÚð ãñ¢ ¥æñÚ ¥æÂ·ðUUU ç×˜æ ·ðUUU Âæâ 5 »éyÕæÚð ãñ¢Ð ¥æÂ·ðUUU ç×˜æ ·ðUUU ×é·UUUæÕÜð ¥æÂ·ðUUU Âæâ ç·UUUUÌÙð »éyÕæÚð ÊØæÎæ ãñ¢?	आपके पास ७ गु4बारे हैं और आपके मित्र के पास ५ गु4बारे हैं। आपके मित्र के मुकाबले आपके पास कितने गु4बारे ज़्यादा हैं?				2				
+			7012	word_problem	°·UUUU àææ¹æ ÂÚ 4 Âÿæè ÕñÆð ãñ¢Ð 1 ©Ç¸ »ØæÐ àææ¹æ ÂÚ ç·UUUUÌÙð Âÿæè Õ¿ð ãñ¢?	एक पेड़ की शाखा पर ८ पक्षी बैठे हैं। कुछ उड़ गए। अब शाखा पर ३ पक्षी हैं। कितने पक्षी उड़ गए?				3				
```

Comment:
Chanakya text and Hindi text are not matching. Some parts are missing in Chanakya text.

Reply:
Resolved